api/volume.go

```diff
@@ -145,6 +145,15 @@ func (s *Server) VolumeAttach(rw http.ResponseWriter, req *http.Request) error {
 	}
 
 	id := mux.Vars(req)["name"]
+	// check attach node state
+	node, err := s.m.GetNode(input.HostID)
+	if err != nil {
+		return err
+	}
+	readyCondition := types.GetNodeConditionFromStatus(node.Status, types.NodeConditionTypeReady)
+	if readyCondition.Status != types.ConditionStatusTrue {
+		return fmt.Errorf("ERROR: Node %v is unschedulable, couldn't attach volume %v to it ", node.Name, id)
```

Comment:
1. `Errorf` is `ERROR` level. No need for `ERROR` again.
2. Unschedulable means replica cannot be scheduled for this node. But if the node is up, we can still attach volumes to it. Error message should be e.g. `node is not ready`.